File: InsertionSort.java
public class InsertionSort {
    public static void main(String[] args) {
        int arr[] = {5,6,8,3,1,6};

        for(int i=1; i<arr.length; i++){
            int temp = arr[i];
            int j = i-1;

            while(j>=0 && arr[j]>temp){

                arr[j+1] = arr[j];
                j--;
            }

            arr[j+1] = temp;

        }
        for (int num : arr) {
            System.out.print(num + " ");            
        }
    }
    
}

File: InsertionSort.py
nums =  [5,6,8,3,1,6]
n = len(nums)
for i in range(1, n):
    temp = nums[i]
    j=i-1
    while j>=0 and nums[j]>temp:
        nums[j+1] = nums[j]
        j-=1

    nums[j+1] = temp
    print(nums)